Cvalidparen/Cvalidparen.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
For the purposes of this repository, `Cvalidparen.yml` and `Cvalidparen.rb` are not needed and should be removed